### ISSUE TITLE
[StratCon][49.19] Integrated command woes

### ISSUE THREAD

**Issue by Tecmes** [MegaMek/mekhq#4100]
There are a couple tickets about specific issues with integrated command bugs, but not about the basic issue, it seems: it doesn't make use of assigned Role.
You can easily end up with heavy lances assigned to pursuits or scanning, etc.
It also gleefully assigns incomplete lances - that are not assignd a Role in the Briefing Room as main lances

StratCon DOES use the Briefing Room roles, at least for scout and reinforcements, according to the FAQ, so it should be doable?

I understand, though, that fitting Roles to specific scenario types is either difficult and/or not coded, but then why not just allow choosing what lance to send, maybe with more stringent limits?

**Comment by Thom293** [comment]
Yeah anything in TOE can be chose as seed lance, even if unassigned.  And dropships it will assign even if not in your TOE.  It would be nice if there was a non-combat role that had a 5% chance of battle, though. (That probably needs an RFE).  

Regarding roles, to my knowledge it has never respected those for certain mission eligibility- it only uses them for their "special ability" (scout surrounding hexes, infantry/mines, reinforce without new roll).   

I think the rational is that integrated is the worst type of contact with least control.  Thus it's limitation.   

And I'm pretty sure a lot of one of your prior RFE is in the FAQ. ;)
